coordinator/src/services/spine-communication.ts

```diff
@@ -81,6 +81,7 @@ export class RequestHandler {
           }
         }
         default: {
+          console.log(`Got the following response from spine:\n${result.data}`)
```

Comment:
Might want to do the same on line 131 for the polling endpoint?

Reply:
Possibly, although if its not a 200 or 202, and all the error conditions are handled by the error handler, what situation would this be handling?